lib/courses/widgets/course_detail_screen.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
```

Comment:
Sorry for being pedantic here, but a newline after the other packages and the schulcloud packages would be nice. See [Schul-Cloud Flutter Code Style Guide](https://github.com/schul-cloud/schulcloud-flutter/wiki/Code-style).